src/pages/applicants/index.tsx

```diff
@@ -179,7 +189,7 @@ const Applicants: NextPage = () => {
                 </PageText>
               </section>
 
-              <section id='application-types'>
+              <section id='application-types' ref={ref3}>
```

Comment:
This section element should include the different application types (which are in the `<Center>` element down below it).

![image](https://user-images.githubusercontent.com/468158/154965791-0da59e61-aa07-4d6d-932a-97d0464e42bb.png)

If you scroll a bit more down from that section, you will see that the selection in the sidebar disappears.

Reply:
good catch, fixed!